x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -201,6 +225,11 @@ export interface HostIsolationResponse {
   action: string;
 }
 
+export type RunningProcessesRequestBody = TypeOf<typeof NoParametersRequestSchema.body>;
+export interface RunningProcessesResponse {
+  action: string;
```

Comment:
Suggestion: for API requests/responses include the word `Api` in it, like: `RunningProcessesApiRequestBody` + `RunningProcessesApiResponse`


Also, I see that we have `ResponseActionApiResponse` type below. I would convert that to a Generic and use it to define each of the response action responses - like:

```typescript
export interface ResponseActionApiResponse<
  TParams extends EndpointActionDataParameterTypes = never,
  TOutput extends object = object> {
  action?: string;
  data: ActionDetails<TParams, TOutput>
}
```

And now:

```typescript
export type RunningProcessesResponse<never, {
    entries: RunningProcessesEntry[];
  }>
```